xbmc/pvr/PVRGUIDirectory.cpp

```diff
@@ -419,65 +421,66 @@ bool CPVRGUIDirectory::GetChannelsDirectory(CFileItemList& results) const
 
 namespace
 {
-  bool GetTimersRootDirectory(const CPVRTimersPath& path,
-                              const std::vector<std::shared_ptr<CPVRTimerInfoTag>>& timers,
-                              CFileItemList& results)
-  {
-    std::shared_ptr<CFileItem> item(new CFileItem(CPVRTimersPath::PATH_ADDTIMER, false));
-    item->SetLabel(g_localizeStrings.Get(19026)); // "Add timer..."
-    item->SetLabelPreformatted(true);
-    item->SetSpecialSort(SortSpecialOnTop);
-    item->SetIconImage("DefaultTVShows.png");
-    results.Add(item);
 
-    bool bRadio = path.IsRadio();
-    bool bRules = path.IsRules();
+bool GetTimersRootDirectory(const CPVRTimersPath& path,
+                            const std::vector<std::shared_ptr<CPVRTimerInfoTag>>& timers,
+                            CFileItemList& results)
+{
+  std::shared_ptr<CFileItem> item(new CFileItem(CPVRTimersPath::PATH_ADDTIMER, false));
+  item->SetLabel(g_localizeStrings.Get(19026)); // "Add timer..."
+  item->SetLabelPreformatted(true);
+  item->SetSpecialSort(SortSpecialOnTop);
+  item->SetIconImage("DefaultTVShows.png");
```

Comment:
can you use setArt("icon") since it's going to be removed anyway?

Reply:
Not really, as this is an unrelated change. I'd prefer if you handle this change in your PR.